Taskfile.yml

```diff
@@ -158,3 +159,21 @@ tasks:
     desc: Accept all test snapshots. (Shortcut to accept snapshots after running tests)
     cmds:
       - Rscript -e 'testthat::snapshot_accept()'
+
+  fmt:
+    desc: Format all R and Rust code.
+    sources:
+      - "{{.R_SOURCE}}"
+      - "{{.R_TESTS}}"
+      - "{{.RUST_SOURCE}}"
+      # These files are auto-generated and ignored by the formatter
+      - exclude: R/000-wrappers.R
+      - exclude: R/import-standalone-*.R
+    generates:
+      - "{{.R_SOURCE}}"
+      - "{{.R_TESTS}}"
+      - exclude: R/000-wrappers.R
+      - exclude: R/import-standalone-*.R
+    cmds:
+      - air format .
+      - cargo fmt --all --manifest-path src/rust/Cargo.toml
```

Comment:
I'd rather have a single `task fmt` for both R and Rust

Reply:
Since deps are generally executed in parallel, it is better to split the task than to execute them sequentially in cmds.

Reply:
Ok but can we still have a `task fmt` that calls `task fmt-r` and `task fmt-rust`?

Reply:
I've cherry picked 66eee85, and will add tasks to format Rust files and format all files

Reply:
Both of those are extremely fast anyway so running sequentially doesn't change much

Reply:
Yeah, just another problem is the task dependencies: since the R files and Rust files are each dependencies of a completely different subsequent tasks, it makes more sense to format them separately than to format them at the same time.

Reply:
Done in 4d9aa0ef771877262ddbf5a773c5989d1f861fb1.

Reply:
Thanks, I think it can be merged now